docs/site/Validation-controller-layer.md

```diff
@@ -0,0 +1,128 @@
+---
+lang: en
+title: 'Validation in the Controller Layer'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation-controller-layer.html
+---
+
+In the case where validation rules are not static, validations cannot be
+specified at the model level. Hence, validation can be added in the controller
+layer.
+
+Take an example of a promo code in an order, it is usually a defined value that
+only valid for a certain period of time. And in the CoffeeShop example, the area
+code of a phone number usually depends on the geolocation.
+
+## Add validation function in the Controller method
+
+The simplest way is to apply the validation function in the controller method,
+that is:
+
+```ts
+// create a validatePhoneNum function and call it here
+if (!this.validatePhoneNum(coffeeShop.phoneNum, coffeeShop.city))
+  throw new Error('Area code in phone number and city do not match.');
+return this.coffeeShopRepository.create(coffeeShop);
+```
+
+## Add interceptor for validation
+
+Another way is to use [interceptors](Interceptors.md).
+
+> Interceptors are reusable functions to provide aspect-oriented logic around
```

Comment:
Remove the `>` characters

Reply:
I'm quoting the definition of interceptors in the docs. but can remove it.